specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/Clusters.json

```diff
@@ -0,0 +1,668 @@
+          ],
+          "description": "Resource tags."
+        }
+      },
+      "description": "The top level Log Analytics cluster resource container.",
+      "x-ms-azure-resource": true
+    },
+    "Cluster": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
+        },
+        "sku": {
+          "$ref": "#/definitions/ClusterSku",
+          "description": "The sku properties."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterProperties",
+          "description": "Log Analytics cluster properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "The top level Log Analytics cluster resource container."
+    },
+    "ClusterListResult": {
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          },
+          "description": "A list of Log Analytics clusters."
+        }
+      },
+      "description": "The list clusters operation response."
+    },
+    "keyVaultProperties": {
+      "type": "object",
+      "description": "The key vault properties.",
+      "properties": {
+        "keyVaultUri": {
+          "description": "The Key Vault uri which holds they key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The version of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyRsaSize": {
+          "description": "Selected key minimum required size.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "billingType": {
+      "type": "string",
+      "readOnly": false,
+      "description": "Configures whether billing will be only on the cluster or each workspace will be billed by its proportional use. This does not change the overall billing, only how it will be distributed. Default value is 'Cluster'",
+      "enum": [
+        "Cluster",
+        "Workspaces"
+      ],
+      "x-ms-enum": {
+        "name": "BillingType",
+        "modelAsString": true
+      }
+    },
+    "ClusterSku": {
+      "type": "object",
+      "description": "The cluster sku definition.",
+      "properties": {
+        "capacity": {
+          "description": "The capacity reservation level in GB for this cluster.",
```

Comment:
>GB [](http://example.com/codeflow?start=60&length=2)

Nit consider being explicit about Gigabytes and not gibibytes.

Reply:
Done.